sos/plugins/__init__.py

```diff
@@ -569,7 +569,7 @@ def add_copy_spec(self, copyspecs, sizelimit=None, tailit=True):
         a single file the file will be tailed to meet sizelimit. If the first
         file in a glob is too large it will be tailed to meet the sizelimit.
         """
-        sizelimit = sizelimit or self.get_option("log_size")
+        sizelimit = sizelimit if sizelimit is not None else self.get_option("log_size")
```

Comment:
This isn't going to work as a single line, as it exceeds the pep8 length limit. We can either stuff `get_option("log_size")` into a local, or rework it as a simple branched `if: ... else:`.